api.go

```diff
@@ -830,3 +834,105 @@ func (c Client) makeTargetURL(bucketName, objectName, bucketLocation string, que
 
 	return u, nil
 }
+
+// isVirtualHostedStyle - returns true if server supports
+// virtual hosted-style urls
+func (c *Client) isVirtualHostedStyle(bucketName string) bool {
```

Comment:
Isn't 'probeVirtualHostStyle' name better ?

Reply:
@vadmeste, you are right. Changed this.

Reply:
> @vadmeste, you are right. Changed this.

Sorry @poornas, I didn't mean this to be changed to probeVirtualHostStyle, I wanted to say setVirtualHostStyle() to be renamed instead. I think this way is still better, what do you think ?